### PR TITLE
[Merged by Bors] - feat: port Data.Prop.PProd

### PR DESCRIPTION
**Pull request by pechersky**
Doesn't need the full port of `logic.basic` so we can port without depending on the other PR

### CODE REVIEW

File: Mathlib/Data/Prod/PProd.lean [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+theorem forall' {p : α → β → Prop} : (∀ x : PProd α β, p x.1 x.2) ↔ ∀ a b, p a b :=
+  PProd.forall
+
+theorem exists' {p : α → β → Prop} : (∃ x : PProd α β, p x.1 x.2) ↔ ∃ a b, p a b :=
+  PProd.exists
+
+end PProd
+
+theorem Function.Injective.pprod_map {f : α → β} {g : γ → δ} (hf : injective f) (hg : injective g) :
```
Comment (semorrison) [inline Mathlib/Data/Prod/PProd.lean:39]:
```suggestion
theorem Function.injective.pprod_map {f : α → β} {g : γ → δ} (hf : injective f) (hg : injective g) :
```

Comment (semorrison) [reply Mathlib/Data/Prod/PProd.lean:39]: I guess this then needs an #align, too.

Comment (pechersky) [reply Mathlib/Data/Prod/PProd.lean:39]: Ah, thanks for catching that. Yes, many files I've now edited, I've done the change by hand. Should the `#align` go in a new PR, for `injective` and `surjective`?

Comment (pechersky) [reply Mathlib/Data/Prod/PProd.lean:39]: #498

**Comment by semorrison** [comment]
bors merge

**Comment by semorrison** [comment]
(I've updated the port-status page.)

File: Mathlib/Data/Prod/PProd.lean [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+theorem forall' {p : α → β → Prop} : (∀ x : PProd α β, p x.1 x.2) ↔ ∀ a b, p a b :=
+  PProd.forall
+
+theorem exists' {p : α → β → Prop} : (∃ x : PProd α β, p x.1 x.2) ↔ ∃ a b, p a b :=
+  PProd.exists
+
+end PProd
+
+theorem Function.injective.pprod_map {f : α → β} {g : γ → δ} (hf : injective f) (hg : injective g) :
```
Comment (semorrison) [inline Mathlib/Data/Prod/PProd.lean:39]:
```suggestion
theorem Function.Injective.pprod_map {f : α → β} {g : γ → δ} (hf : injective f) (hg : injective g) :
```

Comment (pechersky) [reply Mathlib/Data/Prod/PProd.lean]: Sorry, I don't understand -- I thought we're keeping it as `def injective` lowercase.

Comment (pechersky) [reply Mathlib/Data/Prod/PProd.lean]: I think we will need to change the casing on the `hf` and `hg` also.

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**